cosmos/azure-spring-data-cosmos/cosmos-aad-sample/README.md

```diff
@@ -0,0 +1,132 @@
+- (Optional) Maven.
+
+SLF4J is only needed if you plan to use logging, please also download an SLF4J binding which will link the SLF4J API with the logging implementation of your choice. See the [SLF4J user manual](http://www.slf4j.org/manual.html) for more information.
+
+### Clone this repo
+
+1. git clone https://github.com/Azure-Samples/azure-spring-boot-samples.git
+2. cd cosmos/azure-spring-data-cosmos/cosmos-aad-sample
+
+### Create an Azure AD application and service principal
+
+1. Following the instructions [here](https://learn.microsoft.com/azure/active-directory/develop/howto-create-service-principal-portal) for creating an Azure AD application and service principal.
+
+1. In the [authentication](https://learn.microsoft.com/azure/active-directory/develop/howto-create-service-principal-portal#authentication-two-options) section, be sure to select **option 2** to create a new application secret, and make sure you store the secret value somewhere in a text editor. 
+
+1. Search for your app in Azure Portal --> Azure Active Directory --> App Registrations. You should see information like the below:
+
+    ![app](./media/aad-app.png?raw=true "aad app")
+
+1. Review `resources/application.properties` in the repo you have cloned. 
+    - Replace `<Cosmos URI>` with the URI of your Cosmos DB account
+    - Replace `<tenantId>` with `Directory (tenant) ID` from the portal. 
+    - Replace `<clientId>` with `Application (client) ID` from the portal
+    - Replace `<clientSecret>` with the application secret value you created earlier
+    - For the value of `cosmos.defaultScope` replace the `<cosmos account>` part with the name of your Cosmos DB account (note this is used to test connection to AAD)
+    
+### Configure RBAC for your Cosmos DB account
+
+Next we need to create a role that can access your Cosmos DB account appropriately. You should refer to the full instructions [here](https://learn.microsoft.com/azure/cosmos-db/how-to-setup-rbac) for the various options. We'll just keep things simple here by creating a custom role using Azure CLI that has full permissions.
+
+1. First, create a JSON file called `role-definition-rw.json` that contains the following:
+
+    ```json
+    {
+        "RoleName": "MyReadWriteRole",
+        "Type": "CustomRole",
+        "AssignableScopes": ["/"],
+        "Permissions": [{
+            "DataActions": [
+                "Microsoft.DocumentDB/databaseAccounts/readMetadata",
+                "Microsoft.DocumentDB/databaseAccounts/sqlDatabases/containers/items/*",
+                "Microsoft.DocumentDB/databaseAccounts/sqlDatabases/containers/*"
+            ]
+        }]
+    }
+    ```
+ 
+1. Upload the JSON file to Azure CLI, then run the below to create the role, replacing `<myResourceGroup>` and `<myCosmosAccount>` with the resource group name and Cosmos DB account name respectively:
```

Comment:
I think this can be left as it. Please see how this renders in the page/html when viewing. It displays as 1.2.3 etc. 9so one does not have to keep track of ordering when making changes etc0.

Reply:
OK. It's not a problem in the displayed page.